ci.md

```diff
@@ -0,0 +1,201 @@
+# Ignition Continuous Integration
+
+When developers push code changes into Ignition, a series of automated tests
+run for various platforms. This process is called Continuous Integration, or CI.
+This document covers all the CI jobs that are available for Ignition and how to
+interpret their results.
+
+## Types of checks
+
+The main type of check that is performed by CI is compiling the code and running
+all automated tests. Some jobs may also run linters and other static checkers.
+Finally, we have checks that add labels, check DCO and coverage.
+
+All checks are visible at the bottom of a pull request.
+
+![PR checks](images/PR_checks.png)
+
+We use two platforms to run CI, each of them running different builds.
+
+### Jenkins server
+
+Our Jenkins server is located at https://build.osrfoundation.org/. It runs builds
+for all supported operating systems. It also runs an ABI checker for all stable
```

Comment:
Should we define "stable," since it's referred to so many times? Does stable == release?

Reply:
Good point, I'll link to these docs: https://github.com/ignitionrobotics/docs/blob/master/release.md#type-of-releases